README.md

```diff
@@ -1,31 +1,71 @@
 # discordify
+In this guide you can find exactly how to setup discordify for your rich presence needs.
 
 There's a pre-built binary under releases for Windows. You just have to edit the `discordify.exe.config` with the client id's and the Spotify secret.
 
-## Setting up
+## Prerequisites
+In order to use discordify, you'll need to prepare some things before getting started with discorify itself. You'll need to make both a Discord App and a Spotify App. This can be complicated to figure out yourself, but if you follow the steps below you'll be guaranteed to get everything working correctly.
 
-*  Clone the repository `git clone https://github.com/Krognol/discordify` and build the project with visual studio. (Might work with Omnisharp for VScode).
-*  Create an application [at the discord dev pages](https://discordapp.com/developers/applications/me).
-* * You can name it whatever, copy the `Client ID` and set that in `App.config:discord_client_id` .
-* * Click on `Enable rich presence` at the bottom of the page.
-* * Upload a large asset for an icon and name it `spotify_large`.
-* * You'll also want to get the `discord-rpc-w32.dll` from the `lib` dir or from [the official discord-rpc repo](https://github.com/discordapp/discord-rpc).
-* Create a Spotify api application, and copy the Client ID and secret and set that in `App.config:spotify_client_id/secret`
-* Launch discordify and click on `Initialize`
-* Go to your settings in Discord, and go to Games
-* Click on `Add it!` next to `Not seeing your game?`
-* Click on `Discordify.exe`
-* Done!
+### Spotify App
+We'll start off with the easier of the two. We need to get Spotify Client ID and secret, so follow these steps to get those setup for yourself.
+
+* Go to [the Spotify developer dashboard](https://beta.developer.spotify.com/dashboard/) and log in with your Spotify account.
+* Click `Create an App`, followed by `No`. We're creating a non-commercial application.
+* Enter an application name and description. People in Discord won't see this so make it anything you'd like.
+* Click the 3 checkboxes and click `Create`.
+* On the newly created application's page, copy the Client ID and save it for later.
+* On the same page, click `Show Client Secret` and copy the Client Secret and save it for later.
+* You're done!
+
+### Discord App
+This is a little harder than the Spotify App. However, it'll be easy if you just follow these instructs! Let's get started right away.
+
+* Go to [the Discord developer portal](https://discordapp.com/developers/applications/me/create) and log in with your Discord account.
+* Enter an application name. This will be visible as the "game" you're currently playing, so make it something good. A good example would simply be "Spotify".
```

Comment:
From some investigation it may actually be necessary to name the application a little differently. E.g `Spotify.` . Naming it just `Spotify` might be colliding with the actual Spotify process name, and thus does not show the actual rich embed. 

Add that as a notice.